recipe/meta.yaml

```diff
@@ -240,7 +236,7 @@ outputs:
         - python
         - {{ pin_subpackage('ray-tune', exact=True) }}
         - dm-tree
-        - gymnasium ==0.26.3
+        - gymnasium ==0.28.1
         - lz4
         - pyyaml
         - scikit-image
```

Comment:
Could you enable `ray-serve`? fastapi and starlette are available. https://github.com/ray-project/ray/blob/ray-2.37.0/python/setup.py#L270-L275
And remove `aiorwlock` there.

Reply:
A new `ray-cpp` subpackage should be added if possible. https://github.com/ray-project/ray/blob/ray-2.37.0/python/setup.py#L297

Reply:
Yeah, we can avoid adding it here too.